src/main/java/com/openshift/internal/client/httpclient/UrlConnectionHttpClient.java

```diff
@@ -74,6 +76,8 @@
 	protected Integer configTimeout;
 	private String excludedSSLCipherRegex;
 
+	private IAuthorizationStrategy authStrategy;
+
 	public UrlConnectionHttpClient(
 			String username, String password, String userAgent, String acceptedMediaType, String version) {
```

Comment:
shouldnt username and password disappear now that we have an authorization strategy?